lib/xpi/utils.js

```diff
@@ -75,34 +77,24 @@ function createFallbacks(options) {
     console.log("Creating fallbacks if they are necessary..");
   }
 
-  return getManifest({
-    addonDir: options.addonDir
-  }).then(function(manifest) {
+  return getManifest({addonDir: options.addonDir}).then(function(manifest) {
```

Comment:
are you rebased on the latest master? I think this indentation change was needed for the newly added lint checks (probably for line length).